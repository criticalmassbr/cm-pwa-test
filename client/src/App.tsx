import './App.css';
import Home from './pages/Home';

import { BrowserRouter as Router, Route, Link } from "react-router-dom";
import Profile from './pages/Profile';
import { AppProvider } from './hooks';

function App() {

  return (
    <div className="App">
      <AppProvider>
        <Router>
          <div>
            <Route path="/" exact component={Home} />
            <Route path="/me/:id" component={Profile} />
          </div>
        </Router>
      </AppProvider>
    </div>
  );
}

export default App;
