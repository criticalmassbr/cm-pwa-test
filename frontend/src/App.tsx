import React, {useState} from 'react';
import { ApolloClient, InMemoryCache } from '@apollo/client';
import { ApolloProvider } from '@apollo/client/react';
import GlobalStyle from './styles/global';
import { BrowserRouter, Route, Switch } from 'react-router-dom';
import ListUser from './page/ListUser'
import UserDetail from './page/UserDetail'
import Header from './page/Header'
const client = new ApolloClient({
  uri: 'http://127.0.0.1:4000/graphql',
  cache: new InMemoryCache()
});

const App: React.FC = () => {
  const [search, setSearch] = useState('');
  return (
    <BrowserRouter>
    <ApolloProvider client={client}>
      <div>
      <Header search={search} setSearch={setSearch} />
      <Switch>
        <Route path="/" exact children={<ListUser search={search} setSearch={setSearch} />}/>
        <Route path="/user/:id" children={<UserDetail search={search} setSearch={setSearch} />}/>
      </Switch>
      </div>
      
    </ApolloProvider>
    <GlobalStyle />
    </BrowserRouter>
  );
};
export default App;
