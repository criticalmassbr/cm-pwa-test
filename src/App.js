import React from 'react';
import { hot } from 'react-hot-loader/root';

function App() {
  return (
    <div className="App">
      <span>teste</span>
    </div>
  );
}

export default hot(App);
