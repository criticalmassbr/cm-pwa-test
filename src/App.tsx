import React, { useState } from 'react';
import { ThemeProvider } from 'styled-components';
import Query from './components/Query/Query';
import Search from './components/Search';
import { Container, GlobalStyle, Header, SearchSection, Title } from './styles';

const theme = {
  main: 'mediumseagreen',
};

function App() {
  const [artist, setTerm] = useState('');
  const _handleSearch = (term: string) => {
    setTerm(term);
  };

  return (
    <ThemeProvider theme={theme}>
      <GlobalStyle />
      <Container>
        <Header>
          <Title>Album Finder</Title>
        </Header>
        <SearchSection>
          <Search handleSearch={_handleSearch} />
        </SearchSection>
        <Query artist={artist} />
      </Container>
    </ThemeProvider>
  );
}

export default App;
