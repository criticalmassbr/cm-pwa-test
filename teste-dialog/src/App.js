import React from "react";
import { Router } from "react-router-dom";
import GlobalStyle from "./styles/global";

import { Provider } from "react-redux";

import Routes from "./routes";
import Header from "./Components/Header";

import "./config/reactotronConfig";
import history from "./services/history";
import store from "./store";

function App() {
  return (
    <Provider store={store}>
      <Router history={history}>
        <Header />
        <Routes />
        <GlobalStyle />
      </Router>
    </Provider>
  );
}

export default App;
