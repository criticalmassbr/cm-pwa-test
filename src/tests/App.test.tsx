import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

test('renders properly', () => {
  const component = <App />;
  const albumComponent = renderer.create(component);
  const tree = albumComponent.toJSON();
  expect(tree).toMatchSnapshot();
});
