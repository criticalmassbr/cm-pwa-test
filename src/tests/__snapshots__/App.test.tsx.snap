// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  className="sc-fznZeY itetmQ"
>
  <header
    className="sc-fzokOt kDMjiF"
  >
    <div
      className="sc-fzqBZW kiftEj"
    >
      Album Finder
    </div>
  </header>
  <nav
    className="sc-fzqNJr gZyKwe"
  >
    <div
      className="sc-fzpans jnUApY"
    >
      <div
        className="sc-fzplWN chkmDT"
      >
        <svg
          aria-hidden="true"
          className="sc-AxmLO kDecRh sc-fznyAO ckGJxL"
          fill="currentColor"
          focusable="false"
          height="1.5em"
          viewBox="0 0 24 24"
          width="1.5em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M10 18a7.952 7.952 0 004.897-1.688l4.396 4.396 1.414-1.414-4.396-4.396A7.952 7.952 0 0018 10c0-4.411-3.589-8-8-8s-8 3.589-8 8 3.589 8 8 8zm0-14c3.309 0 6 2.691 6 6s-2.691 6-6 6-6-2.691-6-6 2.691-6 6-6z"
          />
        </svg>
        <input
          className="sc-fznKkj fStrHd"
          name="search"
          onChange={[Function]}
          onMouseEnter={[Function]}
          placeholder="Search"
          type="text"
        />
      </div>
    </div>
  </nav>
  <div
    className="sc-AxhCb iwJefs"
  >
    
    
  </div>
  <main
    className="sc-Axmtr jTnQgh"
  />
</div>
`;
